An API incompatibility was encountered while executing org.jbake:jbake-maven-plugin:2.7.0-rc.6:generate: java.lang.UnsatisfiedLinkError

When running the maven plugin on Mac OS (12.6 with Apple Silicon M1 Pro) I encounter the following error

```
[ERROR] Failed to execute goal org.jbake:jbake-maven-plugin:2.7.0-rc.6:generate (default-generate) on project sling-site: Execution default-generate of goal org.jbake:jbake-maven-plugin:2.7.0-rc.6:generate failed: An API incompatibility was encountered while executing org.jbake:jbake-maven-plugin:2.7.0-rc.6:generate: java.lang.UnsatisfiedLinkError: java.lang.UnsatisfiedLinkError: Unable to execute or load jffi binary stub from `/var/folders/f8/_3c0rvgd00n324sqjc8y_3wr0000gn/T/`. Set `TMPDIR` or Java property `java.io.tmpdir` to a read/write path that is not mounted "noexec".
[ERROR] Can't load library: /Users/konradwindszus/git/sling/sling-site/jffi15806522402867392943.dylib
[ERROR] 	at com.kenai.jffi.internal.StubLoader.tempLoadError(StubLoader.java:448)
[ERROR] 	at com.kenai.jffi.internal.StubLoader.loadFromJar(StubLoader.java:433)
[ERROR] 	at com.kenai.jffi.internal.StubLoader.load(StubLoader.java:300)
[ERROR] 	at com.kenai.jffi.internal.StubLoader.<clinit>(StubLoader.java:511)
[ERROR] 	at java.base/java.lang.Class.forName0(Native Method)
[ERROR] 	at java.base/java.lang.Class.forName(Class.java:398)
[ERROR] 	at com.kenai.jffi.Init.load(Init.java:68)
[ERROR] 	at com.kenai.jffi.Foreign$InstanceHolder.getInstanceHolder(Foreign.java:50)
[ERROR] 	at com.kenai.jffi.Foreign$InstanceHolder.<clinit>(Foreign.java:46)
[ERROR] 	at com.kenai.jffi.Foreign.getInstance(Foreign.java:104)
[ERROR] 	at com.kenai.jffi.Platform$ArchHolder.determineCPU(Platform.java:210)
[ERROR] 	at com.kenai.jffi.Platform$ArchHolder.<clinit>(Platform.java:197)
[ERROR] 	at com.kenai.jffi.Platform.getCPU(Platform.java:311)
[ERROR] 	at com.kenai.jffi.Platform.addressMask(Platform.java:345)
[ERROR] 	at com.kenai.jffi.MemoryIO.<clinit>(MemoryIO.java:46)
[ERROR] 	at com.orientechnologies.common.directmemory.OPointer.createNativeBuffer(OPointer.java:57)
[ERROR] 	at com.orientechnologies.common.directmemory.OPointer.getNativeByteBuffer(OPointer.java:31)
[ERROR] 	at com.orientechnologies.common.directmemory.OByteBufferPool.acquireDirect(OByteBufferPool.java:145)
[ERROR] 	at com.orientechnologies.orient.core.storage.memory.ODirectMemoryOnlyDiskCache$MemoryFile.addNewPage(ODirectMemoryOnlyDiskCache.java:501)
[ERROR] 	at com.orientechnologies.orient.core.storage.memory.ODirectMemoryOnlyDiskCache$MemoryFile.access$200(ODirectMemoryOnlyDiskCache.java:463)
[ERROR] 	at com.orientechnologies.orient.core.storage.memory.ODirectMemoryOnlyDiskCache.allocateNewPage(ODirectMemoryOnlyDiskCache.java:245)
[ERROR] 	at com.orientechnologies.orient.core.storage.impl.local.paginated.atomicoperations.OAtomicOperationBinaryTracking.commitChanges(OAtomicOperationBinaryTracking.java:534)
[ERROR] 	at com.orientechnologies.orient.core.storage.impl.local.paginated.atomicoperations.OAtomicOperationsManager.endAtomicOperation(OAtomicOperationsManager.java:338)
[ERROR] 	at com.orientechnologies.orient.core.storage.impl.local.paginated.atomicoperations.OAtomicOperationsManager.executeInsideAtomicOperation(OAtomicOperationsManager.java:165)
[ERROR] 	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.create(OAbstractPaginatedStorage.java:667)
[ERROR] 	at com.orientechnologies.orient.core.db.OrientDBEmbedded.internalCreate(OrientDBEmbedded.java:768)
[ERROR] 	at com.orientechnologies.orient.core.db.OrientDBEmbedded.create(OrientDBEmbedded.java:640)
[ERROR] 	at com.orientechnologies.orient.core.db.OrientDB.createIfNotExists(OrientDB.java:269)
[ERROR] 	at com.orientechnologies.orient.core.db.OrientDB.createIfNotExists(OrientDB.java:256)
[ERROR] 	at org.jbake.app.ContentStore.startup(ContentStore.java:95)
[ERROR] 	at org.jbake.app.Oven.bake(Oven.java:160)
[ERROR] 	at org.jbake.maven.GenerateMojo.reRender(GenerateMojo.java:81)
[ERROR] 	at org.jbake.maven.GenerateMojo.executeInternal(GenerateMojo.java:74)
[ERROR] 	at org.jbake.maven.GenerateMojo.execute(GenerateMojo.java:70)

```

--- a/gradle.properties
+++ b/gradle.properties
@@ -7,7 +7,7 @@ issues                      = https://github.com/jbake-org/jbake/issues
 vcs                         = https://github.com/jbake-org/jbake/
 
 # runtime dependencies
-asciidoctorjVersion         = 2.5.2
+asciidoctorjVersion         = 2.5.7
 asciidoctorjDiagramVersion  = 2.2.1
 args4jVersion               = 2.33
 commonsIoVersion            = 2.11.0

